src/main/webapp/dev/js/common/const.es6

```diff
@@ -58,6 +58,7 @@ const ParamsNames = {
     FEEDBACK_QUESTION_DISCARDCHANGES: 'questiondiscardchanges',
     FEEDBACK_QUESTION_SAVECHANGESTEXT: 'questionsavechangestext',
     FEEDBACK_SESSION_ENABLE_EDIT: 'editsessiondetails',
+    TEAMMATES_EMAIL_ADDRESS: 'teammates@comp.nus.edu.sg',
```

Comment:
Just noticed that this should likely go under the `Const` object below, as `ParamsNames` is used for `name` attributes for inputs that will be used by the backend.